### ISSUE TITLE
Remove Public Enabled is not working

### ISSUE THREAD

**Issue by cjmyers** [SynBioHub/synbiohub3#701]
Remove Public Enabled on the theme page is not displaying correctly and is not being set correctly.

**Comment by cjmyers** [comment]
The button is not saving, but the functionality is fine in SBH1.  The button should make the remove option show up on Manage Collections.

**Comment by cjmyers** [comment]
Maybe rephrase on screen to "Enable Removal of Public Collections"

**Comment by danielfang97** [comment]
Closed with #821
